exotica_python/src/pyexotica/interactive_rho_tuning.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import print_function
+import os
+from time import time
+import Tkinter as tk
+import rospkg as rp
+
+__all__=['Tuning']
+
+class Tuning(object):
+
+    def __init__(self, problem):
+
+        # Initial setup
+        self.master = tk.Tk()
+        self.problem = problem
+
+        # Set title
+        self.master.winfo_toplevel().title("Rho Tuning")
+
+        # Set icon
+        # note, rp.RosPack().get_path('exotica') throws error
+        icon = rp.RosPack().get_path('exotica_core')[:-13] + '/exotica/doc/images/EXOTica_icon.png'
```

Comment:
I don't like this workaround. It has some assumptions on exotica_core being next to exotica, as in our source builds. This may not hold. I have posted a fix using RosStack above.